### ISSUE TITLE
Question about LDCT-IQA and Window/Level Settings

### ISSUE THREAD

**Issue by ning-yuan** [eeulig/ldct-benchmark#6]
Thank you for your great work and for making this implementation open-source!

I have a question regarding the ldct-iqa. During testing, I noticed that the IQA score for the ldose chest images is 0, which I believe might be incorrect.

Additionally, in general, different anatomical regions typically require different window width and window level settings. However, I observed that the same settings were used across regions in the code, which I assume are optimized for chest images. Is this a reasonable approach, or would it be better to adapt the settings for different regions?
![image](https://github.com/user-attachments/assets/72151c7c-d226-42e9-bc8e-b3fb80703f1d)
![image](https://github.com/user-attachments/assets/2b541afe-ed46-4be2-9488-78841fb6e4bf)

Thank you for your insights!

**Comment by eeulig** [comment]
Thanks for your interest in the library! Your results are as expected (we actually show the same table [here](https://eeulig.github.io/ldct-benchmark/dev/examples/ldct_iqa/#evaluate-denoising-methods-using-the-ldctiqa-model) in the documentation). I can also reproduce your results on the full-dose scan of C121.

The main problem with applying this model to chest scans is that it was trained only on abdominal images (please see the warning box *Use with out-of-distribution (OOD) data* in the [documentation](https://eeulig.github.io/ldct-benchmark/dev/examples/ldct_iqa/) and also the paper of the challenge[^1]. Unfortunately there is not much we can do about this as the challenge only contained abdominal images and we don't have reader scores for any other anatomies to train the model on.

Regarding the window settings: We chose to use the same window settings ((C,W)=(-300, 1400)) that were used by the challenge organizers to normalize the (abdominal) training, validation, and test data. The paper[^1] also shows some generalizability experiments in Sec. 3.3 and we couldn't find any statement that other window settings were used for these experiments. We could add an optional parameter to the [evaluate_dicom](https://eeulig.github.io/ldct-benchmark/dev/reference/ldctbench/evaluate/ldct_iqa/#ldctbench.evaluate.ldct_iqa.evaluate_dicom) function to allow the user to set custom windowing settings (this doesn't change the fact that this will always be an OOD setting simply because the network has never seen chest data during training). I tried out evaluating the full-dose scan of C121 with (C,W) = (-600, 1500) (common window for lung exams and actually not too far off from the windowing used for the training data) and get a slightly higher (but still bad) average score of 1.45. I would generally not recommend to use this model to evaluate any non-abdominal scan.

[^1]: Lee, Wonkyeong, Fabian Wagner, Adrian Galdran, Yongyi Shi, Wenjun Xia, Ge Wang, Xuanqin Mou, et al. 2025. “Low-Dose Computed Tomography Perceptual Image Quality Assessment.” Medical Image Analysis 99 (January):103343. <https://doi.org/10.1016/j.media.2024.103343>.
